tests/e2e/source_test.go

```diff
@@ -110,7 +110,8 @@ var _ = Describe("odoSourceE2e", func() {
 		})
 
 		It("Should be able to deploy a dotnet source application", func() {
-			runCmdShouldPass("odo create dotnet dotnet-app --local " + sourceExamples + "/dotnet/")
+			dotnetExampledir := "dotnet"
```

Comment:
Unnecessary use of extra variable. Please remove it and keep the test scenario as it was before